src/domain/_common/components/Avatar/index.tsx

```diff
@@ -0,0 +1,71 @@
+import { ReactComponent as AvatarIcon } from '@/assets/avatar-icon.svg';
+import generateRandomColor from '@/utils/generateRandomColor';
+
+interface AvatarProfileProps {
+  imageUrl?: string;
+  isSmall?: boolean;
+  userName?: string;
```

Comment:
```suggestion
  username?: string;
```
nit. username이라는 단어가 따로 있어서 한 단어로 쓸 수도 있다고합니다~